frontend-project/src/pages/letters/list/index.tsx

```diff
@@ -24,6 +24,11 @@ const TableList: FC<{}> = () => {
       render: (direction: string) =>
         formatMessage({ id: `letters-list.table.direction.${direction.toLowerCase()}` }),
     },
+    {
+      title: formatMessage({ id: 'letters-list.table.columns.date.title' }),
+      dataIndex: 'date',
+      render: (date: string) => date.toLocaleString(),
```

Comment:
Właśnie przetetowałem to na polskiej i angielskiej wersji i zawsze dostaję `6/30/2020, 12:09:06 AM`. Trzeba się zastanowić, co (jak) chcielibyśmy to wyświetlać.

@SmietankowyImperator jesteś w stanie to ustalić i założyć na to osobne zgłoszenie?

Reply:
Jasne dzis usiądę popoludniu i postaram sie rozpisać